lib/client/lambda-server.ts

```diff
@@ -107,7 +121,7 @@ export class LambdaServer {
                 console.log(JSON.stringify(bodyJSON, null, 2));
             }
 
-            const handlerFunction = this.functionName ? this.functionName : "handler";
+            handlerFunction = handlerFunction != null ? handlerFunction : this.functionName ? this.functionName : "handler";
```

Comment:
(undefined != null) is false though thanks to single '=' in javascript's single != comparison operator ... so I believe the above should work as desired when handlerFunction is either null or undefined.  

I changed it though as the your version is prettier and works too (except when supplied 'falsey' values which doesn't matter here).